systemd/CMakeLists.txt

```diff
@@ -30,31 +30,42 @@ if(WITH_SYSTEMD)
     configure_file(${CMAKE_SOURCE_DIR}/systemd/dlt.service.cmake ${PROJECT_BINARY_DIR}/systemd/dlt.service)
     message( STATUS "Configured systemd unit file:dlt.service" )
 
-    configure_file(${CMAKE_SOURCE_DIR}/systemd/dlt-system.service.cmake ${PROJECT_BINARY_DIR}/systemd/dlt-system.service)
-    message( STATUS "Configured systemd unit file:dlt-system.service" )
+    if(WITH_DLT_SYSTEM)
+        configure_file(${CMAKE_SOURCE_DIR}/systemd/dlt-system.service.cmake ${PROJECT_BINARY_DIR}/systemd/dlt-system.service)
+        message( STATUS "Configured systemd unit file:dlt-system.service" )
+    endif(WITH_DLT_SYSTEM)
 
     if(WITH_DLT_DBUS)
         configure_file(${CMAKE_SOURCE_DIR}/systemd/dlt-dbus.service.cmake ${PROJECT_BINARY_DIR}/systemd/dlt-dbus.service)
         message( STATUS "Configured systemd unit file:dlt-dbus.service" )
     endif(WITH_DLT_DBUS)
 
-    configure_file(${CMAKE_SOURCE_DIR}/systemd/dlt-receive.service.cmake ${PROJECT_BINARY_DIR}/systemd/dlt-receive.service)
-    message( STATUS "Configured systemd unit file:dlt-receive.service" )
+    if(WITH_DLT_CONSOLE)
+        configure_file(${CMAKE_SOURCE_DIR}/systemd/dlt-receive.service.cmake ${PROJECT_BINARY_DIR}/systemd/dlt-receive.service)
+        message( STATUS "Configured systemd unit file:dlt-receive.service" )
+    endif(WITH_DLT_CONSOLE)
 
     configure_file(${CMAKE_SOURCE_DIR}/systemd/dlt-example-user.service.cmake ${PROJECT_BINARY_DIR}/systemd/dlt-example-user.service)
```

Comment:
Please add WITH_DLT_EXAMPLES condition as well.

Reply:
Done.

Reply:
Sorry, actually missed the "configure_file() if" clause and only did the "install() if" at first. Fixed now though.

Perhaps it makes sense to merge the if statements. Will try this in a separate commit.